conductr_cli/sandbox_init.py

```diff
@@ -1,62 +1,102 @@
-from conductr_cli import docker_machine, terminal
+from conductr_cli import docker, docker_machine, terminal
 import logging
+import re
 from subprocess import CalledProcessError
 
 
 def init(args):
     """`sandbox init` command"""
 
     log = logging.getLogger(__name__)
-    vm_name = docker_machine.vm_name()
+
+    try:
+        info = terminal.docker_info()
+        existing_ram, has_sufficient_ram = docker_ram_check(info)
+        existing_cpu, has_sufficient_cpu = docker_cpu_check(info)
+
+        if not has_sufficient_ram or not has_sufficient_cpu:
+            if not has_sufficient_ram:
+                log.warning('Docker has insufficient RAM of {}MiB - please increase to a minimum of {}MiB'
+                            .format(existing_ram, docker.DEFAULT_DOCKER_RAM_SIZE))
+
+            if not has_sufficient_cpu:
+                log.warning('Docker has an insufficient no. of CPUs {} - please increase to a minimum of {} CPUs'
+                            .format(existing_cpu, docker.DEFAULT_DOCKER_CPU_COUNT))
+        else:
+            log.info('Docker already has sufficient RAM and CPUs')
+
+    except (AttributeError, CalledProcessError):
+        pass
+
+    docker_machine_vm_name = docker_machine.vm_name()
     if is_docker_machine_installed():
```

Comment:
We are only checking here if `docker-machine` is installed, but not if it is used. It might be the case that the user uses the `Docker.app` and therefore docker-engine, but still has docker-machine installed. In this case we would configure the docker-machine which would be unnecessary and misleading. This case is the default case when upgrading from Docker toolbox to Docker on Mac.